eloquent/distribution.yaml

```diff
@@ -2697,7 +2698,7 @@ repositories:
       url: https://github.com/ros2-gbp/urdfdom_headers-release.git
       version: 1.0.4-1
     source:
-      test_pull_requests: true
+      test_pull_requests: false
```

Comment:
This is the default so the line can be removed.

Reply:
Removed.